Frameworks/UIKit/StarboardXaml/ApplicationCompositor.cpp

```diff
@@ -59,19 +59,24 @@ extern "C" void RunApplicationMain(Platform::String^ principalClassName,
                                    float windowHeight,
                                    ActivationType activationType,
                                    Platform::Object^ activationArg) {
+    // Return from activation immediatly; push all subsequent app launch work to 
```

Comment:
This seems unrelated to AutoLayout issues, and should be tested really thoroughly to make sure we don't accidentally regress anywhere with app launching